src/tests/functional/plugin/cpu/single_layer_tests/topk.cpp

```diff
@@ -283,6 +283,44 @@ INSTANTIATE_TEST_CASE_P(smoke_TopK_dynamic, TopKLayerCPUTest,
         ::testing::ValuesIn(additionalConfig)),
     TopKLayerCPUTest::getTestCaseName);
 
+std::vector<ov::test::InputShape> inputShapes_bubble_BLK_on_channel_horiz = {
+    {{}, {{2, 2, 2, 2}}},
```

Comment:
For topk_bubble_BLK_on_channel_horiz routine in JIT kernel, though we already have test cases with  top_k = 1, axis_dim = 21, which will test this routine. It is still quite necessary to add this corner case with top_k=1, axis_dim=2, where axis_dim < blk_size.